rpm/0013-Allow-mkdir-for-home-.shared-directory.patch

```diff
@@ -0,0 +1,26 @@
+From ec04f1a66b5a08c0f7861a8c90f73f4bfc407769 Mon Sep 17 00:00:00 2001
+From: Dmitry Okoshkin <d.okoshkin@omp.ru>
+Date: Wed, 9 Jun 2021 20:39:19 +0300
+Subject: [PATCH] Allow mkdir for /home/.shared directory
+
+---
+ src/firejail/fs_mkdir.c | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/src/firejail/fs_mkdir.c b/src/firejail/fs_mkdir.c
+index 0e213f2f..0eb4f233 100644
+--- a/src/firejail/fs_mkdir.c
++++ b/src/firejail/fs_mkdir.c
+@@ -34,7 +34,8 @@ static void check(const char *fname) {
+ 
+ 	if (strncmp(fname, cfg.homedir, strlen(cfg.homedir)) != 0 &&
+ 	    strncmp(fname, "/tmp", 4) != 0 &&
+-	    strncmp(fname, runuser, strlen(runuser)) != 0) {
++	    strncmp(fname, runuser, strlen(runuser)) != 0 &&
++		strncmp(fname, "/home/.shared", 13)) {
```

Comment:
This should follow the same style as the other lines, i.e. one less level of indentation and != 0 at the end.

I wonder if the error message should be adjusted a bit too. This is not in user home, just outside of it.

Reply:
Add `/home/.shared` to error message.

Reply:
You modified the line in the patch so that it will no longer apply. You need to modify the file and generate a new patch instead.

Reply:
Yeah, probably okay with https://github.com/sailfishos/firejail/commit/d99b44c9ce1d17db74bab8810b2362125bb0b8da now.

Reply:
Yes, applies nicely. And coding style looks consistent now. Good.